makefiles/Makefile.devhub

```diff
@@ -2,12 +2,20 @@ GIT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
 USER=$(shell whoami)
 
 STAGING_URL="http://docs-devhub-staging.s3-website-us-east-1.amazonaws.com"
-STAGING_BUCKET=docs-devhub-staging
+STAGING_BUCKET=docs-devhub-stg
```

Comment:
Lets not move the Devhub stuff into our account; I think Jordan's planning on their own bucket-related migrations. We should check with him on that.

Reply:
done